sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -155,7 +155,7 @@ def get_table_access_policy(
                 **kwargs)
         except HttpResponseError as error:
             _process_table_error(error)
-        return {s.id: s.access_policy or AccessPolicy() for s in identifiers}  # pylint: disable=E1125
+        return {s.id: s.access_policy or AccessPolicy(start=None, expiry=None, permission=None) for s in identifiers}
```

Comment:
What's the difference here? Is this because some of these parameters are required in the constructor?

Reply:
I seem to recall that setting these values to None _might_ actually change the structure of the request body....

Reply:
Yes these are required parameters. There are actually two different `AccessPolicy` classes right now, one in the generated code and another [here](https://github.com/seankane-msft/azure-sdk-for-python/blob/update-testing/sdk/tables/azure-data-tables/azure/data/tables/_models.py#L32). The latter inherits from the generated code and all arguments are defaulted to `None`.